docker/docker-compose.yaml

```diff
@@ -97,6 +97,45 @@ services:
     networks:
       - von
 
+  react-holder:
+    image: ghcr.io/digicatapult/veritable-holder:latest
```

Comment:
do we want `latest` or pinned versions? I personally prefer pinned versions but debatable, more maintenance

Reply:
happy with latest but we should update `react-authority` to match